### ISSUE TITLE
Show original color's fg/bg hex values in Contrast ratio

### ISSUE THREAD

**Issue by carmacleod** [GoogleChrome/accessibility-developer-tools#309]
When there's a contrast ratio violation, the Color Contrast ratio section of the Accessibility Properties tab currently shows the fg/bg hex values that could be chosen to pass WCAG 2.0 AA and AAA.
It would be useful to also see the original color's fg/bg hex values, so that they could be copy/pasted into a bug report, for example, or so that the developer can quickly see whether the suggested AA/AAA values require changing the fg or bg color (or both) in order to have sufficient contrast.
This would save the step of having to go look the original hex colors up in the Styles tab.
![contrast ratio colors](https://cloud.githubusercontent.com/assets/3331913/19119312/604a9ad2-8aed-11e6-91bf-bc61ec7d40b4.png)
